src/lavinmq/exchange/default.cr

```diff
@@ -6,6 +6,14 @@ module LavinMQ
       "direct"
     end
 
+    def has_binding?(destination : Queue, routing_key : String, headers : Hash(String, AMQP::Field)?)
+      false
```

Comment:
```suggestion
      routing_key == destination.name
```

Reply:
I guess I did return `false` here because `bind` and `unbind` always raises `"Access refused"`, so in it can't have any binding? But maybe it would make more sense to always return true, since no bindings can be added we don't need to write it to any definition file.

However, will change so `bind`/`unbind` returns `true`/`false` instead of having `has_binding?`.